com/truckmuncher/api/trucks.proto

```diff
@@ -192,6 +213,22 @@ message ServingModeRequest {
 }
 
 message ServingModeResponse {
+    /**
+     * The id of the truck that was reported.
+     */
+    optional string truckId = 1;
+    /**
+     * Value in the range [-90, 90]
+     */
+    optional double latitude = 2;
+    /**
+     * Value in the range [-180, 180]
+     */
+    optional double longitude = 3;
+    /**
+     * The aggregated upvote/downvote score for the truck.
+     */
+    optional int currentScore = 4;
```

Comment:
Is having this info useful/required for the web? I don't think the native clients will use it. We'll have to implement some other mechanism (push or poll) to keep the active trucks fresh while a user is in the app (web or native) since other trucks might be coming and going. So if that has to be built anyhow, is there value in having a subset of the "active trucks data" returned in this request?

As a possible alternative, what do we think of returning a `ActiveTrucksResponse` instead? For now, we could send back the full subset, and in the future could look at implementing a diff mechanism based on etags or something.